components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -278,10 +284,43 @@ class AddonsManagerAdapter(
     }
 
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    internal inner class Section(@StringRes val title: Int)
+    internal data class Section(@StringRes val title: Int)
 
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    internal inner class NotYetSupportedSection(@StringRes val title: Int)
+    internal data class NotYetSupportedSection(@StringRes val title: Int)
+
+    /**
+     * Allows to customize how items should look like.
+     */
+    data class Style(
+        @ColorRes
+        val sectionsTextColor: Int? = null,
+        @ColorRes
+        val addonNameTextColor: Int? = null,
+        @ColorRes
+        val addonSummaryTextColor: Int? = null
+    ) {
+        internal fun maybeSetStatusTextColor(textView: TextView) {
```

Comment:
nit: why maybe?

Reply:
Because the value is only going to be set if the property is not null :)

```kt
sectionsTextColor?.let {
                val color = ContextCompat.getColor(textView.context, it)
                textView.setTextColor(color)
            }
```

Reply:
`setStatusTextColorIfAvailable`? Maybe sounds too vague imo :)

Reply:
I used because we are using the same pattern all over the place in the source code XD

Reply:
I prefer to keep it as we are using the same pattern in other [parts of the source code ](https://github.com/mozilla-mobile/android-components/search?l=Kotlin&p=2&q=maybe) and I will prefer to keep consistency :)